### ISSUE TITLE
fit evaluator score

### ISSUE THREAD

**Issue by Cichloeli** [UKPLab/sentence-transformers#1572]
Hi, I am trying to use the `fit` function with an `evaluator`, according to the [documentation](https://www.sbert.net/docs/training/overview.html) "These evaluators are run periodically during training. Further, they return a score and only the model with the highest score will be stored on disc."
So I am wondering, is there a way that I can print the scores returned by the `evaluator` during training so I can make sure the loss is converging and tune the hyper-parameters based on the printed score.
(or any other way to print losses during training)
Thank you!

**Comment by nreimers** [comment]
Add this at the start of your script:
```
from sentence_transformers import LoggingHandler
import logging


logging.basicConfig(format='%(asctime)s - %(message)s',
                    datefmt='%Y-%m-%d %H:%M:%S',
                    level=logging.INFO,
                    handlers=[LoggingHandler()])
```
